### PR TITLE
Book Library

### PR DESCRIPTION
**Pull request by haraphat01**
This is our first project on Javascript Curriculum, the project is a book library where a user can add a book. Check if a book is read or not, and also a function to delete a book.

Kindly check for your approval.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi Team 👋

**Nice work on this and lovely UI design also. ✨**

- [x] There has been a few changes to the linters config files, refer to [this link](https://github.com/microverseinc/linters-config/tree/master/javascript) and kindly update the following files on your repo accordingly:
- `.eslintrc.json`
- `.github/workflows/linters.yml`

- [x] There are two empty files in your project; `form.html` and `sass/script.js`. Please remove them.

- [x] Also kindly remove the commented code in `index.html`.

- [x] Please replace regular functions with es6 arrow functions except in the constructor function.

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: index.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                                    <td></td>
+                                    <td>
+                                        <div class="switch" >
+                                            <div class="cursor"></div>
+                                        </div>
+                                    </td>
+                                    <td>
+                                        <div class="btn btn-danger btn-sm" id="removeBook">Remove</div>
+                                    </td>
+                                </tr> -->
```
Comment (mikenath223) [inline index.html]: - [x] Kindly remove this commented code. It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).

**Comment by haraphat01** [comment]
The inline comment has been removed, the functions have been changed to an arrow functions.

**Review by shloch — APPROVED** [review]
## Hello @haraphat01 , @tigamadou 

GOOD JOB WITH REVISION FROM FORMER TSE 

Really great job...all the logic holds in place and the app functions smoothly
WHile you get a thumbs up, I think you can understand with me that it's good to respect guidelines. You did so with your code except for function names particularly for this exercises (fortunately you had equivalence)

- We were expecting `render()` function (instead of `addBookToView()` , in combination with `loopsbooks()` for your case) [[requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/render_requirement.png)]
- we were expecting `addBookToLibrary()` (you had `addBook()`) [[requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/book_array_requirement.png)]
I think you can understand that some TSE might miss that on the code, on the search of these

### [optional]
- this point isn't on the guidelines, but it's very important for example if the project is to be in your portfolio: it's about the GUI
If you look at the sample below, you notice that the form goes behind the table. It's not practical. It will be good to address this.

lib

Thats all I got for this project, thumbs up for the hard work
Keep up with that
Happy coding

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi **TEAM**, congratulations on fixing all the comments outlined by the previous TSE's :+1: Based on that and having met all the minimum requirements :slightly_smiling_face: I hereby approve of the project :fireworks: 

Go ahead and merge into development :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](linkedin.com/in/daniel-larbi-addo/)

**Comment by tigamadou** [comment]
Thank you very much, Guys. @mikenath223 @shloch @addod19  👍

**Pull request merged**